tests/e2e/specs/tools/ToolSingleDot.spec.js

```diff
@@ -10,8 +10,7 @@ describe("tools/ToolSingleDot", () => {
 
     cy.get('[data-test="grapher-dots--dot"]')
       .should("have.length", 1)
-      .should("have.attr", "cx", "12")
-      .should("have.attr", "cy", "8");
+      .should("have.attr", "transform", "translate(12, 8)");
```

Comment:
This makes so much more sense than keeping the cx and cy!

Reply:
I do appreciate the removal of cx/cy, especially if they were only used for the tests! I feel a little uneasy about having these tests rely on string equality, though, when the actual data we are interested in checking is integral. That being said, I think this is OK for now, especially since this is just in the tests. We could come up with ways to approach this later if it becomes a problem, or if someone feels inspired to come up with a new way to do this.

Reply:
Yup! Doing some cleanup here where `cx` and `cy` don't have any functionality. https://github.com/calband/calchart-redesign/pull/107#discussion_r544806419

Good point about string vs integer equality on tests, but not much we can do in this case... The best we can do is make sure that the integer -> string conversion is consistent when setting the `transform` property.